dev-util/coverity/coverity-2017.07.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Coverity provides the results of analysis on open source coding projects"
+HOMEPAGE="https://www.coverity.com"
+SRC_URI="amd64? ( cov-analysis-linux64-${PV}.tar.gz )
+	x86? ( cov-analysis-linux-${PV}.tar.gz )"
+
+LICENSE="coverity-PLA"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc"
+
+L10N="ja ko zh-CN"
+for _l in ${L10N} ; do
+	IUSE="${IUSE} l10n_${_l}"
+done
+
+DEPEND=""
+RDEPEND="${DEPEND}"
+
+RESTRICT="fetch mirror bindist"
+
+QA_PREBUILT="opt/coverity/bin/*"
+
+pkg_nofetch() {
+	eerror "Please go to https://scan.coverity.com/download"
+	use amd64 && eerror "and download Linux64"
+	use x86   && eerror "and download Linux32"
+	eerror "then put the file in ${DISTDIR}"
+	use amd64 && eerror "named as cov-analysis-linux64-${PV}.tar.gz"
+	use x86   && eerror "named as cov-analysis-linux-${PV}.tar.gz"
+}
+
+src_unpack() {
+	default
+
+	use amd64 && S="${WORKDIR}/cov-analysis-linux64-${PV}"
+	use x86   && S="${WORKDIR}/cov-analysis-linux-${PV}"
+}
```

Comment:
This doesn't make much sense here. Either modify `S` in the global space, or have `src_unpack()` rename the directory so that you don't have to modify `S`.

Reply:
Fixed in next PR